contractcourt/htlc_timeout_resolver.go

```diff
@@ -1294,13 +1294,13 @@ func (h *htlcTimeoutResolver) resolveTimeoutTxOutput(op wire.OutPoint) error {
 // Launch creates an input based on the details of the outgoing htlc resolution
 // and offers it to the sweeper.
 func (h *htlcTimeoutResolver) Launch() error {
-	if h.launched {
+	if h.launched.Load() {
```

Comment:
Same here re read/write methods to encapsulate.

Reply:
yeah I like it, done!